app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class VotesController < ApplicationController
+
+  def create
+    @review = Review.find(params[:review_id])
+    @review.votes.where(:user_id => current_user.id).destroy_all
```

Comment:
(Create method uses current_user, so if that doesn't exist, it'll break the page)